src/app/Http/Middleware/BackpackErrorViews.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Backpack\CRUD\app\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Str;
+
+class BackpackErrorViews
+{
+    public function handle($request, Closure $next)
+    {
+        if (Str::startsWith($request->path(), config('backpack.base.route_prefix'))) {
+            $themeNamespace = substr(config('backpack.ui.view_namespace'), 0, -2);
+            $themeFallbackNamespace = substr(config('backpack.ui.view_namespace_fallback'), 0, -2);
```

Comment:
`substr()` might not be a reliable way to get the "clean" theme namespace. Because the theme namespace can look like this:
- `backpack.theme-tabler::` - from package
- `backpack.theme-coreuiv4::` - from package
- `backpack.theme-coreuiv2::` - from package
- `admin.theme.` - directly from userland
- `vendor.backpack.theme-custom.` - directly from userland

Reply:
Thanks for the heads up, wasn't ware of the last ones. Will fix accordingly.